### PR TITLE
Nft docs

### PR DESCRIPTION
**Pull request by HenriqueNogara**
Adds a simple documentation page for the NFT pallet

### CODE REVIEW

File: polymesh-docs/primitives/nfts.mdx [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
+title: Non-Fungible Tokens
+description: ''
+# slug: /polymesh-docs/primitives/nfts
+subsite: polymesh-docs
+---
+
+# Non-Fungible Token
```
Comment (F-OBrien) [inline polymesh-docs/primitives/nfts.mdx:8]:
```suggestion
# Non-Fungible Tokens 
```

File: polymesh-docs/primitives/nfts.mdx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+description: ''
+# slug: /polymesh-docs/primitives/nfts
+subsite: polymesh-docs
+---
+
+# Non-Fungible Token 
+
+## Overview
+
+The NFT pallet extends the Asset pallet allowing users to create NFT collections, issue unique tokens and redeem (i.e. burn) existing non-fungible tokens. The documentation for the NFT crate can be found [here](https://docs.polymesh.live/pallet_nft/index.html).
```
Comment (F-OBrien) [inline polymesh-docs/primitives/nfts.mdx:12]:
```suggestion
The Non-Fungible Token (NFT) pallet extends the Asset pallet allowing users to create NFT collections, issue unique tokens and redeem (i.e. burn) existing non-fungible tokens. The documentation for the NFT crate can be found [here](https://docs.polymesh.live/pallet_nft/index.html).
```

File: polymesh-docs/primitives/nfts.mdx [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+## NFT Collection
+
+All non-fungible tokens are linked to a unique NFT collection, which is tied to an [Asset](https://developers.polymesh.network/polymesh-docs/primitives/assets/#asset-creation) and user-defined metadata.
+
+### Creating an NFT Collection
+
+In order to successfully create an NFT collection, the dispatchable [`create_nft_collection`](https://docs.polymesh.live/pallet_nft/struct.Module.html#arguments) must be called, and the following conditions must hold:
+* If the asset already exists, it must be of type `NonFungible` and the caller must have the proper permission for the asset. Otherwise, the `create_nft_collection` dispatchable will also create one non-fungible asset using the values passed as `ticker` and `nft_type`;
```
Comment (F-OBrien) [inline polymesh-docs/primitives/nfts.mdx:21]:
```suggestion
* If the asset already exists, it must be of type `NonFungible` and the caller must have the proper permission for the asset. If the asset does not already exist, the `create_nft_collection` dispatchable will also create one non-fungible asset using the values passed as `ticker` and `nft_type`; 
```

File: polymesh-docs/primitives/nfts.mdx [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+### Creating an NFT Collection
+
+In order to successfully create an NFT collection, the dispatchable [`create_nft_collection`](https://docs.polymesh.live/pallet_nft/struct.Module.html#arguments) must be called, and the following conditions must hold:
+* If the asset already exists, it must be of type `NonFungible` and the caller must have the proper permission for the asset. Otherwise, the `create_nft_collection` dispatchable will also create one non-fungible asset using the values passed as `ticker` and `nft_type`; 
+* This must be the first collection associated to the given `ticker` (i.e. only one collection per ticker is allowed);
+* The number of metadata keys associated to the collection must be less or equal to [MaxNumberOfCollectionKeys](https://docs.polymesh.live/pallet_nft/trait.Config.html#associatedtype.MaxNumberOfCollectionKeys);
+* All metadata keys must be registered. This implies that when using local metadata keys, one must call `create_asset` and `register_asset_metadata_local_type` before calling `create_nft_collection`;
```
Comment (F-OBrien) [inline polymesh-docs/primitives/nfts.mdx:24]:
```suggestion
* All metadata keys must be registered prior to the creation of the collection. This implies that when using local metadata keys, one must call `create_asset` and `register_asset_metadata_local_type` before calling `create_nft_collection`;
```

File: polymesh-docs/primitives/nfts.mdx [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+Once the dispatchable succeeds, a unique ID is tied to the NFT collection.
+
+### Issuing an NFT 
+
+After creating the collection, tokens can be issued with the [`issue_nft`](https://docs.polymesh.live/pallet_nft/struct.Module.html#arguments-1) extrinsic. For successfully issuing an NFT, the following conditions must hold:
+* An NFT collection associated with `ticker` must exist;
+* The caller must have the proper permission for the `Asset`;
+* The `portfolio_kind` of the caller must be valid;
+* All metadata keys specified in the NFT collection must be set when issuing the token. Attempting to issue a token with metadata keys not defined in the collection will also fail;
```
Comment (F-OBrien) [inline polymesh-docs/primitives/nfts.mdx:34]:
```suggestion
* All metadata keys specified in the NFT collection must have a value set when issuing the token. Attempting to issue a token with metadata keys not defined in the collection will also fail;
```

**Review by F-OBrien — CHANGES_REQUESTED** [review]
Minor comments but otherwise LGTM

**Review by adamdossa — APPROVED** [review]

**Pull request merged**